src/pika_binlog_bgworker.cc

```diff
@@ -29,16 +32,17 @@ void BinlogBGWorker::DoBinlogBG(void* arg) {
   Cmd* c_ptr = self->GetCmd(opt);
   if (!cinfo_ptr || !c_ptr) {
     LOG(WARNING) << "Error operation from binlog: " << opt;
+    goto CLEANUP;
```

Comment:
we don't suggest use goto in our program, since goto make the program unreadable..

Reply:
use `goto` to clean up the resources and nested loops shouldn't be harmful, while it would make the code more clean and tidy